### PR TITLE
Ninja cloak is no longer true invisibility

### PR DESCRIPTION
**Pull request by Anewbe**
Alpha = 5
Ling cloak is Alpha = 10

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/modules/ninja.dm [hunk 1]
```diff
@@ -37,8 +37,8 @@
 
 	var/mob/living/carbon/human/H = holder.wearer
 
-	H << "<font color='blue'><b>You are now invisible to normal detection.</b></font>"
-	H.invisibility = INVISIBILITY_LEVEL_TWO
+	to_chat(H, "<font color='blue'><b>You are now invisible to normal detection.</b></font>")
```
Comment (mistyLuminescence) [inline code/modules/clothing/spacesuits/rig/modules/ninja.dm]: Can we change this to "almost invisible", then?

Comment (Anewbe) [reply code/modules/clothing/spacesuits/rig/modules/ninja.dm]: It's good enough as-is.

Comment (mistyLuminescence) [reply code/modules/clothing/spacesuits/rig/modules/ninja.dm]: It's not *true*, though - 5 alpha isn't full invisibility, it's just really, really good camouflage. If you're moving around in a well-lit area, you're still just barely visible to someone who's paying attention.

**Comment by Belsima** [comment]
what the fuck does bikeshed mean

File: code/modules/clothing/spacesuits/rig/modules/ninja.dm (hunk 1 above)
Comment (Belsima) [reply code/modules/clothing/spacesuits/rig/modules/ninja.dm]: Yeah, it's really not invisible, that's misleading. I can totally see a ninja running around at alpha 5 if I'm already aware they exist.

**Comment by elgeonmb** [comment]
This is a good label

**Pull request merged**